moveit_core/exceptions/src/exceptions.cpp

```diff
@@ -35,14 +35,18 @@
 /* Author: Acorn Pooley, Ioan Sucan */
 
 #include <moveit/exceptions/exceptions.h>
-#include <ros/console.h>
+#include <rcutils/logging_macros.h>
+#include "rclcpp/rclcpp.hpp"
+
+// Logger
+rclcpp::Logger logger_exceptions = rclcpp::get_logger("exceptions");
```

Comment:
```suggestion
static const rclcpp::Logger LOGGER = rclcpp::get_logger("exceptions");
```
(see https://github.com/ros-planning/moveit2/pull/33)

Reply:
I'm ok with this change but it'll require further changes in the file (those places where logger_exceptions is used will also need to be renamed).

Reply:
Maybe, shall we leave this change aside for now and make a global change if we decide to use `LOGGER`?

Reply:
I'm fine with this!